libs/designsystem/src/lib/scss/base/_list.scss

```diff
@@ -0,0 +1,86 @@
+@import '../utils';
+
+$marker-disc-font-size: 1.3em;
+$marker-cirle-font-size: 0.8em;
```

Comment:
These values seem a little arbitrary, but I guess that is ok. Is it aligned with the sketch library? They seem to be smaller than the 6px suggested in the sketch library.

Reply:
I actually meant Zeplin, heres a link: https://zpl.io/a8rZN84

Reply:
Can I get access to the Zeplin project mentioned above? @RasmusKjeldgaard

Reply:
I think @bommariusser (Marvin) should be able to give you that.

Reply:
@bommariusser I need access to Zeplin, too